includes/processors/class.llms.processor.membership.bulk.enroll.php

```diff
@@ -168,7 +169,7 @@ public function task( $item ) {
 
 		$query = new LLMS_Student_Query( $item['query_args'] );
 
-		if ( $query->found_results ) {
+		if ( $query->has_results() ) {
```

Comment:
I think that this is probably fine, but a direct replacement for `found_results` is `get_found_results()`

Reply:
In this case, I saw that we weren't using the number of found results, only checking if there were any found results.